src/main/java/pl/javastart/restoffers/category/Category.java

```diff
@@ -0,0 +1,68 @@
+package pl.javastart.restoffers.category;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import pl.javastart.restoffers.offer.Offer;
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    @Column(length = 1000)
+    private String description;
+
+    @JsonIgnore
+    @OneToMany(mappedBy = "category", cascade = CascadeType.REMOVE)
+    private Set<Offer> offers;
```

Comment:
Z tego co widzę to korzystasz z dto, więc obiekt tej klasy nie jest serializowany do Jsona, więc `@JsonIgnore` zbędne